low-level/framework/utils/conf_utils.py

```diff
@@ -148,3 +154,9 @@
 
 SRVNODE = Conf.get(GLOBAL_CONF, f'{CLUSTER}>{SERVER_NODES}')[MACHINE_ID]
 ENCLOSURE = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{STORAGE}>{ENCLOSURE_ID}")
+
+# Get SYSTEM INFORMATION specific config to use in other files.
+GLOBAL_SITE_ID = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{SITE_ID}",'DC01')
```

Comment:
I dont think GLOBAL needs to be prepended, as there is no LOCAL version of it. Just keep SITE_ID & alike for others

Reply:
<SITE_ID> var name is already used above.
#L122 SITE_ID="site_id"
Still changes required?

Reply:
Why this GLOBAL_ seen still ?